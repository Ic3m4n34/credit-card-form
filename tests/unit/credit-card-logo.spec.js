import { mount } from '@vue/test-utils';
import CreditCardLogo from '@/components/credit-card-logo';

describe('CreditCardLogo', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(CreditCardLogo);
  });

  it('renders component', () => {
    expect(wrapper.classes()).toEqual(expect.arrayContaining(['credit-card-logo']));
  });

  it('displays default credit card', () => {

  });
});
