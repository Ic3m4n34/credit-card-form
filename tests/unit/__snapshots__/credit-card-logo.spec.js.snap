// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditCardLogo renders component 1`] = `<div class="credit-card-logo"></div>`;
